tests/utils.py

```diff
@@ -333,3 +337,47 @@ def is_running_in_github_fork() -> bool:
 skipifgithubfork = pytest.mark.skipif(
     is_running_in_github_fork(), reason="Skipping test because it runs on a PR coming from fork"
 )
+
+
+def assert_load_info(info: LoadInfo, expected_load_packages: int = 1) -> None:
```

Comment:
all of them are available in `tests/pipeline`. your choice
1. use existing one
2. move existing one here

Reply:
Good catch, thank you! I'll prefer using the existing ones!